pir/proto/payload.proto

```diff
@@ -43,10 +43,24 @@ message Response {
 
 // Private information retrieval setup parameters
 message PIRParameters {
-    // Number of items in the database
-    uint64 database_size = 1;
+    // Number of items in the database (NOT number of plaintexts)
+    uint64 num_items = 1;
+
+    // Number of plaintexts in the database
+    uint64 num_pt = 4;
```

Comment:
Maybe reorder the indices?

Reply:
We could, since we haven't actually released this code anywhere, but in general it's a bad practice. You want to never change indices of any fields that have ever actually been used.